### PR TITLE
Add on player leave town event

### PR DESCRIPTION
**Pull request by sergix44**
Useful for integration with other plugins

### CODE REVIEW

**Comment by NeumimTo** [comment]
The event is cancellable but you are ignoring event result

**Comment by sergix44** [comment]
> The event is cancellable but you are ignoring event result

You're right, fixed.

File: src/main/java/org/redcastlemedia/multitallented/civs/menus/common/ConfirmationMenu.java [hunk 1]
```diff
@@ -85,10 +87,14 @@ public boolean doActionAndCancel(Civilian civilian, String actionString, ItemSta
                 }
             } else if ("leave".equals(type)) {
                 if (town != null) {
-                    town.getRawPeople().remove(civilian.getUuid());
-                    player.sendMessage(LocaleManager.getInstance().getTranslation(player,
-                            "you-left-town").replace("$1", town.getName()));
-                    TownManager.getInstance().saveTown(town);
+                    PlayerLeaveTownEvent event = new PlayerLeaveTownEvent(civilian.getUuid(), town);
```
Comment (NeumimTo) [inline src/main/java/org/redcastlemedia/multitallented/civs/menus/common/ConfirmationMenu.java:90]: You could pass the civilian object.

Comment (sergix44) [reply src/main/java/org/redcastlemedia/multitallented/civs/menus/common/ConfirmationMenu.java:90]: I've passed the UUID just to keep it consistent with the other events

**Pull request merged**